Changelog.md

```diff
@@ -6,7 +6,7 @@ Language Features:
 Compiler Features:
  * Commandline Interface: Return exit code ``2`` on uncaught exceptions.
  * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
- * EVM: Basic support for the EVM version "Paris".
+ * EVM: Basic support for the EVM version "Paris". Make ``prevrandao`` a valid opcode name that refers to the same opcode as the existing ``difficulty``. Emit the new name when the code targets "Paris" and later EVM versions. The change is due to the renaming introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
```

Comment:
```suggestion
 * EVM: Support for the EVM version "Paris". Introduce ``block.prevrandao`` in Solidity and ``prevrandao()`` in inline assembly, resp. as Yul builtin, to replace ``block.difficulty`` (still allowed with a warning when targetting Paris or later) and ``difficulty()`` (disallowed when targetting Paris or later). Use ``prevrandao`` as opcode name and in evmasm, when the code targets "Paris" and later EVM versions. The change is due to the renaming introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
```
We should probably update this to reflect the current state of the change. I actually think, contrary to earlier suggestions, that it may make sense to split this up into several points like:

- Support for the EVM version "Paris".
- Introduce ``block.prevrandao`` and ``prevrandao()`` in inline assembly for EVM versions >= paris.
- Deprecate ``block.difficulty`` and disallow ``difficulty()`` in inline assembly for EVM versions >= paris.

Or something along those lines.